### PR TITLE
scala@2.12 bumped up to version 2.12.10

### PR DESCRIPTION
**Pull request by saqib-trywe**
- [ x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/scala@2.12.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
-class ScalaAT212 < Formula
+!class ScalaAT212 < Formula
```
Comment (fxcoudert) [inline Formula/scala@2.12.rb]: This change should be removed

File: Formula/scala@2.12.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
   homepage "https://www.scala-lang.org/"
-  url "https://downloads.lightbend.com/scala/2.12.9/scala-2.12.9.tgz"
-  mirror "https://www.scala-lang.org/files/archive/scala-2.12.9.tgz"
-  mirror "https://downloads.typesafe.com/scala/2.12.9/scala-2.12.9.tgz"
-  sha256 "5e9f044f579b60215f144ba21878c9abef3b9541b37b34c1744c436861eef672"
+  url "https://downloads.lightbend.com/scala/2.12.10/scala-2.12.10.tgz"
+  mirror "https://www.scala-lang.org/files/archive/scala-2.12.10.tgz"
+  mirror "https://downloads.typesafe.com/scala/2.12.10/scala-2.12.10.tgz"
+  sha256 "3b12bda3300fedd91f64fc7f9165fd45c58328b1b760af24ca6ffe92e3b0656a"
+
```
Comment (fxcoudert) [inline Formula/scala@2.12.rb]: Extra blank line is not needed.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
Please rebase on top of master, and keep one blank line above `bottle` (you had two blank lines, in your previous version, and zero now)

**Comment by saqibsaleem** [comment]
I have rebased but to no-effect. The conflict remained so I have resolved the conflict in GitHub and merged.

**Pull request merged**